app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,27 @@
+class PostsController < ApplicationController
+  def create
```

Comment:
- [ ] PostsController#index displays posts in timeline
![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/final_project/milestone3/posts2.png)

Let's add the index page for posts.